conf/tachyon-env.sh.template

```diff
@@ -69,22 +64,12 @@ CONF_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
 export TACHYON_JAVA_OPTS+="
   -Dlog4j.configuration=file:$CONF_DIR/log4j.properties
-  -Dtachyon.debug=false
-  -Dtachyon.worker.tieredstore.level.max=1
```

Comment:
Since this is bundled with the following configurations, may be better to leave it here until the following tieredstore configurations go to properties file too.